psst-gui/src/ui/credits.rs

```diff
@@ -0,0 +1,218 @@
+use std::sync::Arc;
+
+use crate::{
+    cmd,
+    data::{AppState, ArtistLink, Nav},
+    ui::theme,
+    ui::utils,
+};
+use druid::{
+    widget::{Controller, CrossAxisAlignment, Flex, Label, List, Maybe, Painter, Scroll},
+    Cursor, Data, Env, Event, EventCtx, Lens, RenderContext, Target, UpdateCtx, Widget, WidgetExt,
+};
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Data, Lens, Deserialize)]
+pub struct TrackCredits {
+    #[serde(rename = "trackUri")]
+    pub track_uri: String,
+    #[serde(rename = "trackTitle")]
+    pub track_title: String,
+    #[serde(rename = "roleCredits")]
+    pub role_credits: Arc<Vec<RoleCredit>>,
+    #[serde(rename = "extendedCredits")]
+    pub extended_credits: Arc<Vec<String>>,
```

Comment:
For this we could use the don't deserd option

Reply:
Ok I made all the possible credits optional I believe.